libyul/optimiser/DataFlowAnalyzer.h

```diff
@@ -134,6 +136,9 @@ class DataFlowAnalyzer: public ASTModifier
 	/// Returns true iff the variable is in scope.
 	bool inScope(YulString _variableName) const;
 
+	/// Returns the literal value of the identifier, if it exists.
```

Comment:
Is this useful to have in `DataflowAnalyzer`?

Reply:
It was initially a lambda inside LoadResolver, but @ekpyron suggested that it's useful to move it inside DataFlowAnalyzer. see: https://github.com/ethereum/solidity/pull/11026#discussion_r594218192

Reply:
I would say that especially the data flow analyzer should be kept as simple as possible, but maybe it's fine to put this function here since it deals with a member of the class.